dispatch/dispatch.go

```diff
@@ -414,6 +459,12 @@ func (ag *aggrGroup) fingerprint() model.Fingerprint {
 	return ag.labels.Fingerprint()
 }
 
+func (ag *aggrGroup) GroupID() string {
+	h := sha1.New()
```

Comment:
Any reason to use sha1? Will xxhash or other libraries works

Reply:
only reason of using sha1 is wanting to keep the output of the hash to be short as this is going to be nexttoken for user to use